src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final TalonFX intakeMotor;
+
+    private final DigitalInput rightBeamBreak;
+    private final DigitalInput leftBeamBreak;
+
+    public IntakeSubsystem() {
+        intakeMotor = new TalonFX(0); 
+        intakeMotor.setNeutralMode(NeutralModeValue.Brake);
+        //intakeMotor.setInverted(true);
+        rightBeamBreak = new DigitalInput(0);
+        leftBeamBreak = new DigitalInput(0);
+    }
+
+    public void setIntakeSpeed(double speed) {
+        intakeMotor.set(speed);
+    }
+
+    public void setOuttakeSpeed(double speed) {
```

Comment:
Is `setOuttakeSpeed` and `stop` necessary? Can we rely on the user to know that  `setIntakeSpeed` takes a value from -1 to 1 and 0 to stop, perhaps through JavaDocs. If so, where would `setOutakeSpeed` and `stop` need to be used?